### ISSUE TITLE
Primary immunodeficiency

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6161]
**Preferred term label:**
Primary immunodeficiency

**Synonyms**
PID

**Definition (free text, please give PubMed ID)**
Defects in the immune system development and/or function. PIDs are broadly classified as disorders of adaptive immunity (i.e., T cell, B-cell or combined immunodeficiencies) or of innate immunity (e.g., phagocyte and complement disorders).; 
PMID: 30275850 

**Parent term (use hpo.jax.org/app)**
Immunodeficiency

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: # 616576

**Comment by pnrobinson** [comment]
This term refers to the cause of immunodeficiency as seen in a group of diseases ("secondary"). The HPO does not encode etiologies, so this is not a good term. See #6162
